src/utilities/valueFromAST.ts

```diff
@@ -148,7 +148,7 @@ export function valueFromAST(
     // no value is returned.
     let result;
     try {
-      result = type.parseLiteral(valueNode, variables);
+      result = type.parseLiteral(valueNode, variables, true);
```

Comment:
Considering `valueFromAST()` deprecated, we don't 100% have to support this new feature, but no harm.

Reply:
```suggestion
      result = type.parseLiteral(valueNode, variables);
```